v3/plugins/redhat/vscode-tekton/0.2.0/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-tekton
+version: 0.2.0
+type: VS Code extension
+displayName: Tekton
+title: Tekton pipelines support
+description: This extension adds rich pipeline support for Tekton
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-tekton
+category: Language
+firstPublicationDate: '2020-07-27'
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-tekton:0.2.0-7a8b4ad"
+      name: vscode-tekton
+      memoryLimit: '384Mi'
+      volumes:
+        - mountPath: "/home/theia/.kube"
```

Comment:
I am ok with that but, i think all the plugins interacting with kubernetes should use the same volume name. So I suggest https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/redhat/vscode-openshift-connector/0.1.5/meta.yaml#L18-L20 to be renamed kube-config instead of oc-config. And adding a volume in the kubernetes plugin with the same name.
BTW any reasons why the kubernetes plugin is having such a high memory request? https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/ms-kubernetes-tools/vscode-kubernetes-tools/1.2.0/meta.yaml#L17

Reply:
Why do they all have to have the same name? The OpenShift connector one is for persisting OpenShift configs, this one is for kube configs. IMO the name should be different to reflect that.

The memory requirement is higher for K8s tooling because it requires it, especially when building containers inside a container (dogfooding).

Reply:
@sunix sometimes even 1Gb is not enough when building containers from big Dockerfiles

Reply:
@sunix can we merge this one now?

Reply:
no ... plz rename the with the same name.
at least we would let the users use the tekton plugin after login with the openshift connector. So i would rename the openshift connector name to match with this one.

Reply:
so `kube` or `kube-config` what ever but name the openshift connector volume with `kube` or `kube-config` so if you do login with the openshift connector and connect to one cluster. you would be able to access to tekton plugin ... but if it doesn't work ... i don't see any reason to make that happen. It is our duty to provide to the users, plugins that would live more than 3 months....

Reply:
Done